.eleventy.js

```diff
@@ -0,0 +1,25 @@
+// `require` the package at the top of the file with all the others
+const embedYouTube = require("eleventy-plugin-youtube-embed");
+
+// 11ty configuration
+module.exports = config => {
+	
+// adds embed plugin
+config.addPlugin(embedYouTube);
+
```

Comment:
the `css` folder needs to move into the `src` folder and `config.addPassthroughCopy('src/css');` needs to be added here

Reply:
Actually it looks like the css folder can be deleted and a new one added to map the fonts